Makefile

```diff
@@ -275,3 +259,16 @@ openshift-image-mirror-mappings:
 	$(CONTAINER_ENGINE) pull registry.ci.openshift.org/ci/promoted-image-governor:latest
 	$(CONTAINER_ENGINE) run --rm -v "$(CURDIR):/release:z" registry.ci.openshift.org/ci/promoted-image-governor:latest --ci-operator-config-path /release/ci-operator/config --release-controller-mirror-config-dir /release/core-services/release-controller/_releases --openshift-mapping-dir /release/core-services/image-mirroring/openshift --openshift-mapping-config /release/core-services/image-mirroring/openshift/_config.yaml
 .PHONY: openshift-image-mirror-mappings
+
+config_updater_vault_secret:
+	@[[ -z $$cluster ]] && echo "ERROR: \$$cluster must be set" && exit 1
+	$(CONTAINER_ENGINE) pull registry.ci.openshift.org/ci/applyconfig:latest
+	$(CONTAINER_ENGINE) run --rm \
+		-v $(CURDIR)/clusters/build-clusters/common:/manifests:z \
+		-v "$(kubeconfig_path):/_kubeconfig:z" \
+		registry.ci.openshift.org/ci/applyconfig:latest \
+		--config-dir=/manifests \
+		--context=$(cluster) \
+		--kubeconfig=/_kubeconfig
+	make hack/ci-secret-generator.sh
```

Comment:
```suggestion
	make ci-secret-generator
```

Reply:
We also need to land the kubeconfig for the new cluster after `applyconfig` and before `ci-secret-generator`

```
oc --context "$(cluster)" sa create-kubeconfig -n ci config-updater > "$(build_farm_credentials_folder)/sa.config-updater.$(cluster).config"
```